acbuild/write.go

```diff
@@ -34,11 +33,10 @@ func init() {
 	cmdAcbuild.AddCommand(cmdWrite)
 
 	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
-	cmdWrite.Flags().BoolVar(&sign, "sign", false, "sign the resulting ACI")
```

Comment:
Should we perhaps be a bit more gentle here, still accepting the old flag and emitting a custom error for this specific case, instructing the user about direct gpg invocation?